test/spec/integration_spec.js

```diff
@@ -0,0 +1,30 @@
+describe('Integration Spec', function(){
+    var rules = customRules;
+
+    beforeEach(function() {
+        jasmine.getFixtures().fixturesPath = '/base/';
+    });
+
+    it('rules can be configured with axe-core', function(){
+        axe.configure(rules);
+        configuredRules = axe.getRules(['edx-custom']);
+        configuredRuleIds = [for (rule of configuredRules) rule.ruleId].sort(); // jshint ignore:line
+        expectedRuleIds = [for (rule of rules.rules) rule.id].sort(); // jshint ignore:line
+        expect(configuredRuleIds).toEqual(expectedRuleIds);
+    });
+
+    it('checks can be configured with axe-core', function(){
+        axe.configure(rules);
+        configuredChecks = axe._audit.checks;
```

Comment:
Same `var` recommendation as above for `configuredRules ` and `configuredRuleIds`.